### PR TITLE
CP-26391: Display the SR-IOV network status on Networking Tab Page

### PR DESCRIPTION
**Pull request by jishengx**
Signed-off-by: Jisheng Xing <jisheng.xing@citrix.com>

### CODE REVIEW

File: XenModel/Messages.resx [hunk 1]
```diff
@@ -13656,4 +13656,7 @@ You will need to navigate to the Console on each of the selected VMs to complete
   <data name="NICS_ARE_SRIOV_ENABLED" xml:space="preserve">
     <value>All the supported NICs have enabled SR-IOV.</value>
   </data>
+  <data name="REQUIRED_HOST_REBOOT" xml:space="preserve">
```
Comment (cheng-z) [inline XenModel/Messages.resx]: These strings like REQUIRED_HOST_REBOOT need to be ordered by alphabet.
This file is too big and shown incorrect in Web Github, what I think it should not add into the end of this file but insert to the file as alphabet.

**Comment by cheng-z** [comment]
Except this comments, all looks good to me.

File: XenModel/Messages.resx (hunk 1 above)
Comment (MihaelaStoica) [reply XenModel/Messages.resx]: Yes, it looks like the file became unsorted at some point. We should make sure that it is alphabetically sorted.

File: XenAdmin/Controls/NetworkingTab/NetworkList.cs [hunk 2]
```diff
@@ -946,6 +948,9 @@ public void UpdateDetails()
                     Pif == null ? Messages.NONE : Pif.LinkStatusString();
                 MacCell.Value = Pif != null && Pif.IsPhysical() ? Pif.MAC : Messages.SPACED_HYPHEN;
                 MtuCell.Value = Network.CanUseJumboFrames() ? Network.MTU.ToString() : Messages.SPACED_HYPHEN;
+                SriovCell.Value = Pif == null || Pif.sriov_logical_PIF_of == null || Pif.sriov_logical_PIF_of.Count == 0
```
Comment (MihaelaStoica) [inline XenAdmin/Controls/NetworkingTab/NetworkList.cs]: As I had also said on PR  #1954, please create a PIF.IsSrIovLogicalPIF() function ans use it here as well.

**Comment by cheng-z** [comment]
Hi @MihaelaStoica 
Could you please help to have a look again and merge if no other concern?
Thanks.

**Comment by MihaelaStoica** [comment]
Waiting for another commit that sorts Messages.rex

**Pull request merged**